clarify README w.r.t. installation and configuration (#7)
costela/docker-volume-hetzner: README.md

## Usage

First, create an API key from the Hetzner Cloud console and save it temporarily.

Install the plugin as described above. Then, set the API key in the plugin options, where `<apikey>` is the key you just created:

```
$ docker plugin set hetzner apikey=<apikey>
```

> The plugin is then ready to be used, e.g. in a Docker Compose file, by setting the `driver` option on the docker `volume` definition (assuming the alias `hetzner` passed during installation above).

Reviewer: ```suggestion
The plugin is then ready to be used, e.g. in a `docker-compose` file, by setting the `driver` option on the docker `volume` definition (assuming the alias `hetzner` passed during installation above).
```

> For example, when using the following Docker Compose volume definition in a project called `foo`:

Reviewer: ```suggestion
For example, when using the following `docker-compose` volume definition in a project called `foo`:
```

- **`apikey`** (**required**): authentication token to use when accessing the Hetzner Cloud API
- **`size`** (optional): size of the volume in GB (default: `10`)
- **`fstype`** (optional): filesystem type to be created on new volumes. Currently supported values are `ext{2,3,4}` and `xfs` (default: `ext4`)

> - **`prefix`** (optional): prefix to use when naming created volumes; it will be separated from the created volume name with a hyphen (default: `docker`)

Reviewer: maybe we could be even clearer with an example?
```suggestion
- **`prefix`** (optional): prefix to use when naming created volumes; the final name on the HC side will be of the form `prefix-name`, where `name` is the volume name assigned by `docker` (default: `docker`)
```

Author: Good suggestion! :+1:
